### PR TITLE
MetricsDownloader Tasks

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

**Comment by imsickofmaps** [comment]
Looks good, except just make sure each task returns something. It helps with logging. Like `return "Created new record <%s>" % newmetric.id`

**Comment by gsvr** [comment]
Happy with what I added?

File: django_refugeerights/metricsdownloader/tasks.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    """
+    Task to look up a metric value from the metricstore
+    """
+    metric_client = MetricsApiClient(settings.METRIC_API_KEY)
+    response = metric_client.get_metric(
+        metric.lookup_name,
+        "-" + settings.METRIC_RECORDING_INTERVAL,
+        settings.METRIC_RECORDING_INTERVAL,
+        "")
+    metric_value = float(response.values()[0][-1]['y'])
```
Comment (imsickofmaps) [inline django_refugeerights/metricsdownloader/tasks.py]: What is there's no results?

Comment (gsvr) [inline django_refugeerights/metricsdownloader/tasks.py]: Hmm, do you think we should record 0.0 or rather not record anything?  I'm leaning towards not recording anything.

Comment (imsickofmaps) [inline django_refugeerights/metricsdownloader/tasks.py]: Record nothing please

**Comment by gsvr** [comment]
@imsickofmaps Please re-review.

**Comment by imsickofmaps** [comment]
:+1:

**Pull request merged**